app/views/buyers/applications/_listing.html.slim

```diff
@@ -99,7 +99,7 @@ table class="data"
           - if cinstance.trial?
             = remaining_trial_days(cinstance)
         - unless @account
-          td = link_to_buyer_or_deleted cinstance.user_account, :admin_buyers_account_path
+          td = link_to_buyer_or_deleted cinstance.account, :admin_buyers_account_path
```

Comment:
I was worried about this change, especially because `AccountContract` repeats `belongs_to :user_account`, but I think it won't be a problem after all. `account` is just an alias for an association of `Contract` in the end.

Reply:
yes, I did it this way just to avoid adding more code, but I could also just add an alias in the decorator to be able to use both names if we want to.